src/main/java/uk/ac/ebi/ena/readtools/cram/Cram2Fastq.java

```diff
@@ -274,14 +264,9 @@ public CollatingDumper(InputStream cramIS, CRAMReferenceSource referenceSource,
 
 		@Override
 		protected AbstractFastqReader newReader() {
```

Comment:
After we remove AbstractFastqReader then the constructor method should be renamed to createMultiFastqOutputter() or similar. Looks very confusing to the average person as we are supposedly creating a 'reader' but are actually creating a fastq 'writer'.